### ISSUE TITLE
Create a "Donate" button similar to the "Register" and the "Athlete's Guide" buttons

### ISSUE THREAD

**Issue by AnaHotaling** [realbig/RBM_TH_EPIC#22]
We would like a new button, Donate, to be created. It should have the same dimensions as the Register and Volunteer buttons and be located in the same area. However, we would like control over the appearance of this button, just as we do for the Athlete's Guide: having it be an option in the template. If it is checked, it shows up. If it is unchecked, it is hidden. Like the Athlete's Guide, there should also be a field in the template where we can input the donation link.

This is a priority; we had a meeting with representatives from Crowdrise this afternoon, and we are planning on launching the Ann Arbor Marathon donation page on Friday, with the other events following shortly after. I've copied Eva's email below, to verify approval from her:

Yes, let's ask for a donate button.  I'm thinking it should be up with the register and volunteer buttons.  What do you think?  Speaking of Race Day Guide- how hard would it be for you to create one for Frosty and Krazy?

Thanks!

**Comment by brashrebel** [comment]
Sure thing. That mostly sounds reasonable. I'll admit that I'm nervous when you say you want control over how it looks but it should be the same size and location as other buttons that match the site's theme. I tend to believe that a _very_ good reason is necessary to break with consistency throughout the site. But maybe that isn't what you're asking for. Can you elaborate more on what control you need exactly?

Other than that, I'd like some clarification on _exactly_ where this new button should appear. There's a few places where Volunteer and Register show up.

The only other issue I see is the timeline. We are committed all of this week and can't start this until mid next week. Is that a serious problem?

**Comment by AnaHotaling** [comment]
Sorry I wasn't clear. We want the same control over the Donate button as we have over the Athlete's Guide button. Currently, the Athlete's Guide button stays hidden until we check the check box on the template. Then it shows up on the event page. If we do not check the check box, it stays hidden. We also control what link the Athlete's Guide button links to on each event page. We want this same functionality with the Donate button. This is because our featured charities will change from event to event and from year to year, so we will need deactivate the button for events whose featured charities are not set up, and we will need to be able to change the link as we change charities. 

Eva would like the Donate button to look exactly like the Register and Volunteer buttons that are at the top of every event page, beneath the event date and time. The Donate button should be to the right of these two buttons.

We did want to have the Donate functionality up for the Ann Arbor Marathon by Friday; the other events can wait until next week. Please let me know if this is a problem. Thanks!

**Comment by brashrebel** [comment]
Thanks for the explanation Ana. I'll see what I can do for timing and this will take ~2 hours of billable time which I'll need Eva's approval for.

**Comment by AnaHotaling** [comment]
I've texted Eva to let her know. My forwarding her email isn't enough for approval? We were both hoping that would do it, since she's tied up in meetings and will be leaving the country next week (and will be hard to get a hold of... and I know we have at least one more change coming, to the home page where the partners used to be).

**Comment by brashrebel** [comment]
I'd love to know about that home page change soon as it is easier for us to work in another task during the same time we're working on the site than fit it in separately.

And I just want to ensure Eva never gets surprised by an invoice. If you are confident of that then we can schedule it.

**Comment by AnaHotaling** [comment]
I'm confident about the Donate button! And I opened the widget issue. :) Thanks!
